crd-generator/api/src/main/java/io/fabric8/crd/generator/AbstractJsonSchema.java

```diff
@@ -78,6 +79,11 @@
   public static final String ANNOTATION_JSON_IGNORE = "com.fasterxml.jackson.annotation.JsonIgnore";
   public static final String ANNOTATION_JSON_ANY_GETTER = "com.fasterxml.jackson.annotation.JsonAnyGetter";
   public static final String ANNOTATION_JSON_ANY_SETTER = "com.fasterxml.jackson.annotation.JsonAnySetter";
+  public static final String ANNOTATION_MIN = "io.fabric8.generator.annotation.Min";
```

Comment:
All annotations should be put in the same package, imo.

Reply:
At this point we have annotations in 2 places:
 - `io.fabric8.generator.annotation` -> generic modifiers to represent CRD properties in Java code
 - `io.fabric8.crd.generator.annotation` -> specific annotations to tweak the specifically the `crd-generator`
 
 Do you see any other invariant @metacosm ?

Reply:
As soon as the annotations are used in different contexts, it would be better for users to help discoverability, in particular, for the annotations to be in the same location even though they initially were developed for different purposes… Obviously, changing annotations location at this point wouldn't be backwards compatible so maybe just a matter of documentation? It might make sense to explain why they live in different packages so that users (and future maintainers, down the line 😸) don't think it's an arbitrary decision.

Reply:
Added a brief JavaDoc on each annotation, let me know if you want other/more changes for this 🙂